assets/templates/partials/header.tmpl

```diff
@@ -244,7 +244,7 @@
             </ul>
         </nav>
     </div>
-    {{if not .SearchDisabled}}
+    <!--{{if not .SearchDisabled}}
```

Comment:
typo ?

Reply:
fixed this, although for now i am leaving the commented code in there as i may still use it